### PR TITLE
first pass at vignette on forecast data

### LINKED ISSUE #20: vignette describing forecast data

### PR DESCRIPTION
**Pull request by elray1**
This would resolve #20

### CODE REVIEW

File: vignettes/forecast_data.Rmd [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+```{r}
+head(forecast_outputs)
+str(forecast_outputs)
+```
+
+This is a data frame with four groups of columns (see the [hubverse documentation](https://hubverse.io/en/latest/user-guide/model-output.html) for more information about these data formats):
+
+1. The `model_id` identifies the model that produced the predictions.
+2. Together, the `location`, `reference_date`, `horizon`, `target_end_date`, and `target` columns serve to identify a prediction task:
```
Comment (lshandross) [inline vignettes/forecast_data.Rmd:49]: Noting that these columns can be called the task IDs may be helpful for consistency across hubverse materials

File: vignettes/forecast_data.Rmd [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+# Subset the forecasts and observations to those that we will plot
+forecasts_to_plot <- forecast_outputs |>
+  filter(
+    model_id == "MOBS-GLEAM_FLUH",
+    target == "wk flu hosp rate",
+    location == "25",
+    reference_date == "2022-12-17"
+  ) |>
+  mutate(output_type_id = as.numeric(output_type_id))
+head(forecasts_to_plot)
```
Comment (lshandross) [inline vignettes/forecast_data.Rmd:144]: It would be useful to be able to see the `output_type_id` and `value` columns here

**Comment by harryhoch** [comment]
A few comments:

1. The first table truncates the content of the target row. It would be good to show the whole thing if possible.
2. With location, we should be clear about the code. What are folks outside of the US supposed to use?

**Comment by elray1** [comment]
Thanks for your suggestions, @lshandross and @harryhoch!  I have made some updates to address these comments.  I'll leave this PR open for a bit (say, till Tuesday) in case you or others have any additional suggestions.

**Pull request merged**